convert.lua

```diff
@@ -168,8 +176,8 @@ function convert_prot.dissector(buffer, pinfo, tree)
         return
```

Comment:
OK, here you check if we already parse the beginning of the connection. If we did, we stop doing anything. That's good!